.github/workflows/build-test

```diff
@@ -33,6 +33,7 @@ ARTIFACTSDIR=${GITHUB_WORKSPACE}/wheelhouse
 rm -rf ${ARTIFACTSDIR} && mkdir ${ARTIFACTSDIR}
 
 python -m pip install --upgrade pip wheel build
+python -m pip install qcs-sdk-python==0.20.1
```

Comment:
The other versions are currently not working. Not sure if we should add this as a dependency or ci is sufficient?

Reply:
I suggest adding it as a dependency if it's required to make the package work.

Reply:
it is already a dependency of pyquil, but only this version works currently.